examples/cluster/playground/Dockerfile

```diff
@@ -1,27 +1,33 @@
-FROM ubuntu:bionic
+FROM ubuntu:jammy
 
-ARG PIP_INDEX_URL
-ENV PIP_INDEX_URL=${PIP_INDEX_URL:-https://pypi.python.org/simple}
+RUN apt-get update -yq && \
+    apt-get install -yq \
+        # needed to add a ppa
+        software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa
 
 RUN apt-get update -q && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
         software-properties-common \
         debhelper dpkg-dev gcc gdebi-core git help2man libffi-dev \
-        libssl-dev libsasl2-modules libyaml-dev pyflakes python3-dev python3-pip python3-pytest \
-        python-tox python-yaml wget zip zsh \
+        dh-virtualenv \
+        libssl-dev libsasl2-modules libyaml-dev pyflakes3 python3.8-dev python3.8-distutils python3-pip python3-pytest python3-http-parser\
+        tox python3-yaml wget zip zsh \
         openssh-server docker.io curl vim jq libsvn-dev \
     && apt-get clean
 
-RUN apt-get update -q && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3.6 python3.6-dev python3.6-venv
-RUN wget https://bootstrap.pypa.io/pip/3.6/get-pip.py
-RUN python3.6 get-pip.py
-RUN ln -s /usr/bin/python3.6 /usr/local/bin/python3
-
-RUN cd /tmp && \
-    wget http://mirrors.kernel.org/ubuntu/pool/universe/d/dh-virtualenv/dh-virtualenv_1.0-1_all.deb && \
-    gdebi -n dh-virtualenv*.deb && \
-    rm dh-virtualenv_*.deb
+ARG PIP_INDEX_URL
+ARG NPM_CONFIG_REGISTRY
+ENV PIP_INDEX_URL=${PIP_INDEX_URL:-https://pypi.python.org/simple}
+ENV NPM_CONFIG_REGISTRY=${NPM_CONFIG_REGISTRY:-https://npm.yelpcorp.com}
+
+# Get yarn
+# I'd use ubuntu yarn (yarnpkg) but https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1019291
```

Comment:
is this still an issue? the linked bug says this should be fixed (which means hopefully we can also get rid of this workaround in tron)